extensions/amp-a4a/0.1/test/test-real-time-config-manager.js

```diff
@@ -376,9 +381,12 @@ describes.realWin('real-time-config-manager', {amp: true}, env => {
         'https://www.2.com',
       ];
       const expectedRtcArray = [
-        {response: rtcCalloutResponses[0], callout: urls[0], rtcTime: 10},
-        {response: rtcCalloutResponses[1], callout: urls[1], rtcTime: 10},
-        {callout: urls[2], error: RTC_ERROR_ENUM.INSECURE_URL, rtcTime: 10},
+        {response: rtcCalloutResponses[0],
+          callout: getCalloutParam(urls[0]), rtcTime: 10},
+        {response: rtcCalloutResponses[1],
+          callout: getCalloutParam(urls[1]), rtcTime: 10},
+        {callout: getCalloutParam(urls[2]),
+          error: RTC_ERROR_ENUM.INSECURE_URL, rtcTime: 10},
       ];
       return executeTest({
         urls, inflatedUrls: urls, rtcCalloutResponses, calloutCount,
```

Comment:
You also need explicit tests of `getCalloutParam`: I think these tests would all pass if it just always returned `foo`.